--- a/engines/director/configure.engine
+++ b/engines/director/configure.engine
@@ -1,3 +1,3 @@
 # This file is included from the main "configure" script
 # add_engine [name] [desc] [build-by-default] [subengines] [base games] [deps] [components]
-add_engine director "Macromedia Director" yes "" "" "highres" "imgui indeo3 qdm2 svq1"
+add_engine director "Macromedia Director" yes "" "" "highres" "imgui indeo3 qdm2 svq1 cdtoons enet fmtowns_pc98_audio gif hnm indeo45 jyv1 lua mfc vpx theoradec midi mpeg2 sid_audio tinygl truemotion1 universaltracker vgmtrans_audio xan"
--- a/configure
+++ b/configure
@@ -4534,7 +4534,7 @@
 # Some platforms (eg. Android, iOS) may use an edited version
 # of SDL-net or SDL2-net that does not require SDL or SDL2 respectively
 #
-if test "$_sdlnet" = auto ; then
+if test "$_sdlnet" != no ; then
 	# If SDL3 was detected, then test for SDL3_net exclusively
 	# If SDL2 was detected, then test for SDL2_net exclusively
 	# If SDL was detected, then test for SDL_net exclusively
